src/main/java/com/team4/testingsystem/controllers/QuestionController.java

```diff
@@ -34,6 +37,22 @@ public QuestionDTO getQuestion(@PathVariable("id") Long id) {
         return questionConverter.convertToDTO(questionService.getById(id));
     }
 
+    @ApiOperation(value = "Get questions from the database by it's level id")
+    @GetMapping("/level/{levelId}")
+    public List<QuestionDTO> getQuestionsByLevel(@PathVariable("levelId") Long levelId) {
+        return questionService.getQuestionsByLevelId(levelId).stream()
+                .map(questionConverter::convertToDTO)
+                .collect(Collectors.toList());
+    }
+
+    @ApiOperation(value = "Get questions from the database by it's module id")
+    @GetMapping("/module/{moduleId}")
+    public List<QuestionDTO> getQuestionsByModule(@PathVariable("moduleId") Long moduleId) {
+        return questionService.getQuestionsByModuleId(moduleId).stream()
+                .map(questionConverter::convertToDTO)
+                .collect(Collectors.toList());
+    }
+
```

Comment:
Don't forget that the expected result for listening is quite different. I would suggest creating a special endpoint for it